File: Practice/21/Python/example21.py



def bmi(weight: float, height: float):
	height = height / 100
	bmitog = weight / height**2
	return bmitog

def print_bmi(bmi: float):
	if (bmi < 18.5) :
		print("Underweight")
	
	elif (bmi >= 18.5 and bmi < 25):
		print("Normal")
	
	elif (bmi >= 25 and bmi < 30) :
		print("Overweight")
	
	elif (bmi > 30) :
		print("Obesity")
	
weight,height = map(str, input().split())
print_bmi(bmi(float(weight), float(height)))

File: Practice/21/C++/example21/example21/example21.cpp
#include <iostream>
#include <string>
using namespace std;


double BMI(double weight, double height) {
	height = height / 100;
	double bmitog = weight / pow(height, 2);
	return bmitog;
}
void printBMI(double BMI) {
	if (BMI < 18.5) {
		cout << "Underweight";
	}
	else if (BMI >= 18.5 && BMI < 25) {
		cout << "Normal";
	}
	else if (BMI >= 25 && BMI < 30) {
		cout << "Overweight";
	}
	else if (BMI > 30) {
		cout << "Obesity";
	}
}


int main()
{
	double weight, height;
	cin >> weight >> height;
	printBMI(BMI(weight, height));
}